src/main/java/com/exasol/releasedroid/usecases/repository/ReleaseLetter.java

```diff
@@ -87,6 +90,22 @@ public String getFileName() {
         return this.ticketNumbers;
     }
 
+    /**
+     * Get summary section.
+     * 
+     * @return summary section if exists
+     */
+    public Optional<String> getSummary() {
+        final int start = this.body.indexOf("## Summary");
+        final int end = this.body.indexOf("#", start + 2);
```

Comment:
The summary could also have a subheading or simply contain a #. For example `#exasol` ;-)

Reply:
It shouldn't...

Reply:
I don't believe. Prove by a test ;-)